model/notification/db.go

```diff
@@ -71,11 +72,14 @@ func (n *Notification) SetBSON(raw mgobson.Raw) error {
 		n.Payload = &message.GithubStatus{}
 
 	case event.GithubMergeSubscriberType:
-		n.Payload = &commitqueue.GithubMergePR{}
+		n.Payload = &commitqueue.GitHubMergePR{}
```

Comment:
I see why you changed this to be capital but I disagree this is necessary--we use lowercase github everywhere else so this is inconsistent (see line 74 haha)

Reply:
removed